owncloudDomain/src/test/java/com/owncloud/android/domain/files/usecases/CopyFileUseCaseTest.kt

```diff
@@ -109,4 +112,30 @@ class CopyFileUseCaseTest {
 
         verify(exactly = 1) { repository.copyFile(any(), any()) }
     }
+
+    @Test
+    fun `should return list of files when repository returns it`() {
```

Comment:
The convention we follow here for tests name is: `<action> - ok/ko (depends if it is a correct case or not) - <brief description>`. Let's try to follow it so that we have a consistent naming through the tests 👍

Reply:
Ok I change the name for maintain the consistency in the test. But I think with the naming convention we are using the purpose of the test is less clear.